pkg/web_app/lib/src/search.dart

```diff
@@ -16,6 +16,103 @@ void setupSearch() {
   _setEventsForSearchForm();
   _setEventForFiltersToggle();
   _setEventForSortControl();
+  _setEventForScreenshot();
+}
+
+void _setEventForScreenshot() {
```

Comment:
I think this feature should go into its own file, because here we are handling events that trigger the search, and not about the display.

Reply:
done.